pastepwn/analyzers/tests/mailpasswordanalyzer.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import unittest
+from unittest import mock
+
+from pastepwn.analyzers.mailpasswordanalyzer import MailPasswordAnalyzer
```

Comment:
It should be possible to do `from pastepwn.analyzers import MailPasswordAnalyzer if you add the analyzer to the [package init file](https://github.com/d-Rickyy-b/pastepwn/blob/master/pastepwn/analyzers/__init__.py)